cookbooks/anisotropic_viscosity/av_material.h

```diff
@@ -0,0 +1,50 @@
+/*
+  Copyright (C) 2015 - 2024 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _aspect_material_model_AV_h
+#define _aspect_material_model_AV_h
+
+namespace aspect
+{
+  namespace HeatingModel
+  {
+    template <int dim>
+    class ShearHeatingAnisotropicViscosity : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
+    {
+      public:
+        /**
+         * Compute the heating model outputs for this class.
+         */
+        void
+        evaluate (const MaterialModel::MaterialModelInputs<dim> &material_model_inputs,
+                  const MaterialModel::MaterialModelOutputs<dim> &material_model_outputs,
+                  HeatingModel::HeatingModelOutputs &heating_model_outputs) const override;
+
+        /**
+         * Allow the heating model to attach additional material model outputs.
+         */
+        void
+        create_additional_material_model_outputs(MaterialModel::MaterialModelOutputs<dim> &material_model_outputs) const override;
+    };
+  }
```

Comment:
With the changes in #6728 you can now also delete this part. The shear heating plugin for anisotropic viscosity is now a normal heating plugin that you can activate in the `Heating model` subsection using `set List of model names = anisotropic shear heating`.
So in essence I dont think this header file is still necessary at all.